frontend/app/services/MemberService.scala

```diff
@@ -128,25 +128,18 @@ class MemberService(identityService: IdentityService,
         case Failure(e) => logger.error(s"Could not create paid Zuora subscription: ID=${user.id}", e)
       }
 
-    def updateSalesforceContactWithMembership(stripeCustomer: Option[Customer]): Future[ContactId] =
-      salesforceService.updateMemberStatus(user.minimal, tier, stripeCustomer).andThen { case Failure(e) =>
```

Comment:
Ah, it looks like this is how the Stripe Customer makes it onto the contact. @paulbrown1982 - I think you wanted to remove this anyway?

Reply:
We need to check if there are any SF triggers requiring this to take place during the upgrade flow.

Reply:
Yes, @MichaelOjoGIT raised a few concerns with this, so I'll do some further testing and then discuss with @davidpepper223 before merging...